backend/authentication/auth-handler.js

```diff
@@ -74,7 +74,31 @@ module.exports.authController = async (event, context, callback) => {
             }
             break;
         }
-    //---------- LIST ALL ADMINS ----------
+        //---------- ADMIN LOGIN ----------
+        case 'adminLogin': {
```

Comment:
Please, split this file to do just a single task like Registration, Logging In, or Listing the Admins, but not all of them.